### PR TITLE
updated to 7.0.0.Final

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: download/download.adoc [hunk 1]
```diff
@@ -52,44 +42,6 @@
 |===
 
 
-== Latest development version:  #{site.pom.latest.version}
```
Comment (psiroky) [inline download/download.adoc]: I think we have latest development version: 7.1.0.Beta1?

**Comment by mbiarnes** [comment]
yepp - but we decided last time when releasing CAP1 or 7.1.0.Beta1 noy to upload the artifacts/binaries to filemgmt.jboss.org. So when the next release will be done tomorrow we can upload the binaries of 7.1.0.Beta2 and then we have artifact for this development version. Otherwise I have to download the artifact from Nexus and upload them to filemgmt.jboss.org

**Comment by psiroky** [comment]
So we have the artifacts in Maven Central, but not on downloads.jboss.org? That sounds weird. In any case, if the files are not there and no is looking for them we can leave it be.

**Comment by mbiarnes** [comment]
I asked if I should upload the binaries to filemgmt - and I was told not do on this case as this is a special case with CAP1, I released it and I released it with the name 7.1.0.Beta1. So when you release the binaries will get sonner or later to Maven Central - whereas they were not uploaded (manually or via script) to filemgmt.jboss.org. Yes  - a kind of weird but thta's the situation. So I expect we should wait and upload the artifact of CAP2 (7.1.0.Beta2) release.

**Comment by mbiarnes** [comment]
@psiroky  7.1.0.Beta2 added as development version

**Pull request merged**